### DISCUSSION TITLE
Conversation filtering would be nice

### DISCUSSION THREAD

**Discussion by FlsPrpht** [community/community#13023] [Pull Requests]
When there are lots of comments, it would be nice to be able to filter based on who made the coomment and on resolved/not resolved.

**Comment by George-Swartzendruber-FHR** [comment]
Not an answer, but wanted to second this. In addition, being able to filter down to just comments would be helpful when there are lots of commit messages as well.

**Comment by tal-sapan** [comment]
It would be helpful to filter conversations according to file, for example when you review new commits and want to check that all the conversations open on the file you are reviewing were addressed.
